source/conf/Configuration.cpp

```diff
@@ -53,7 +53,11 @@ namespace conf
 	std::unique_ptr<Albinos::Config, DestroyConfig> tmp;
 	{
 	  Albinos::Config *tmp2 = nullptr;
-	  Albinos::createConfig("feathers", &tmp2);
+	  if (Albinos::createConfig("feathers", &tmp2) != Albinos::SUCCESS)
```

Comment:
What happens when we bail out ? Does the compositor just exit ?

Reply:
We juste don't configure anything.

Reply:
(feathers will still run)